Examples/HelloWorld/CSharp/Program.cs

```diff
@@ -4,6 +4,10 @@
 
 using ArrayFire;
 
+// If using Visual Studio 2015 you can uncomment the following lines and type Sin() instead of Arith.Sin(), Seq() instead of Util.Seq(), and so on.
+// using static ArrayFire.Arith;
+// using static ArrayFire.Util;
+
```

Comment:
Can you add a Device.Info() after SetBackend?
Also use Backend.DEFAULT as the argument to set Backend.